content/soarian/dstu2.md

```diff
@@ -113,6 +122,20 @@ ID                                | Value\[x] Type      | Description
 <%= json(:SOARIAN_CUSTOM_EXTENSION) %>
 
 
+## Pagination
+
+The pagination links are included in the Bundle when a resource returns multiple items. It is important to follow these Link header values instead of constructing your own URLs.
+
+<%= json(:SOARIAN_LINK_EXTENSION) %>
```

Comment:
Is this really an extension or is it using the standard links in the bundle?

Reply:
(in examples, it looks normal, so the naming of this with "extension" is confusing - could also just link to the FHIR doc that discusses what these links mean)

Reply:
You're right. i think naming them "SOARIAN_FHIR_LINKS"  would make more sense. Any suggestions for other nomenclature is welcome.

Reply:
Modified link example URL and changes are present here https://github.com/cerner/fhir.cerner.com/pull/232/commits/dbec31a2c6140761e3a20ba9d91a4b8858c141bc